packages/kit/test/apps/basics/test/server.test.js

```diff
@@ -330,7 +330,7 @@ test.describe('Load', () => {
 			await page.goto(`/load/fetch-origin-external?port=${port}`);
 			expect(await page.textContent('h1')).toBe(`origin: ${new URL(baseURL).origin}`);
 		} finally {
-			await page.close();
+			await context.close();
 			close();
```

Comment:
is this line missing an await?

Reply:
mhm good question - @benmccann does this make a difference? (same happens in another test that was changed here)

Reply:
I don't know. It's an interesting question. We should be consistent one way or the other though: https://github.com/sveltejs/kit/pull/8447